packages/components/src/components/domainproperties/issues/IssuesListDefPropertiesPanelFormElements.tsx

```diff
@@ -201,6 +231,7 @@ export class AssignedToGroupInput extends React.PureComponent<AssignmentOptionsI
     }
 
     onChange = (name: string, formValue: any, selected: Principal, ref: any): any => {
```

Comment:
return type for this function can be changed from `any` to `void`

Reply:
updated